src/app/public/modules/file-attachment/file-attachment.component.spec.ts

```diff
@@ -1083,6 +1083,14 @@ describe('File attachment', () => {
       expect(fixture.nativeElement).toBeAccessible();
     });
   }));
+
+  it('should pass accessibility when label changes', async(() => {
+    fixture.componentInstance.labelText = 'Something different';
+    fixture.detectChanges();
+    fixture.whenStable().then(() => {
+      expect(fixture.nativeElement).toBeAccessible();
```

Comment:
We should now be using `expectAsync` on these.

Reply:
See https://github.com/blackbaud/skyux-indicators/blob/65ba36456b769183a1102842438302340307d9e2/src/app/public/modules/wait/wait.component.spec.ts#L50

Reply:
Done.